Sources/UIEvent+fromSDL_Event.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  UIEvent+fromSDL_Event.swift
+//  UIKit
+//
+//  Created by Geordie Jay on 05.11.18.
+//  Copyright © 2018 flowkey. All rights reserved.
+//
+
+#if os(Android)
```

Comment:
is this the current pattern for android only files? We could exclude them from the iOS and mac targets

Reply:
Sounds good to me, thanks